### PR TITLE
try push to org

### PR DESCRIPTION
**Pull request by Kholoudnour**
test pull request

### CODE REVIEW

File: database/migrations/2024_08_25_033712_create_categories_table.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('categories', function (Blueprint $table) {
+    
+        $table->id();
+        $table->string('name')->unique();
+        $table->text('description')->nullable();
```
Comment (Nora-Lootfy) [inline database/migrations/2024_08_25_033712_create_categories_table.php:18]: i don't think there is a description for a category, it is just a name and you may add an image

File: database/factories/CategoryFactory.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'id' => fake()->num(),
+            'name' => fake()->name(),
```
Comment (Nora-Lootfy) [inline database/factories/CategoryFactory.php:21]: i think this will fall

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
you have to modify categories migration file make it consistent with model and factory

File: database/migrations/2024_08_25_033712_create_categories_table.php (hunk 1 above)
Comment (Kholoudnour) [reply database/migrations/2024_08_25_033712_create_categories_table.php:18]: do I add another table for the image?

File: database/factories/CategoryFactory.php (hunk 2 above)
Comment (Kholoudnour) [reply database/factories/CategoryFactory.php:21]: why? why?

File: database/migrations/2024_08_25_033712_create_categories_table.php (hunk 1 above)
Comment (Basma-Abdallah) [reply database/migrations/2024_08_25_033712_create_categories_table.php:18]: at the same table insert an image for each category as displayed in the website

File: database/factories/CategoryFactory.php (hunk 2 above)
Comment (Basma-Abdallah) [reply database/factories/CategoryFactory.php:21]: because id is a PK ? @Nora-Lootfy

**Pull request merged**